.babelrc.js

```diff
@@ -12,5 +12,6 @@ module.exports = {
   plugins: [
     '@babel/plugin-syntax-dynamic-import',
     ['@babel/plugin-proposal-object-rest-spread', { useBuiltIns: true }],
+    ['@babel/plugin-proposal-optional-chaining', { loose: true }],
```

Comment:
This line should be removed because it's not yet supported by ESLint (I would very much wish it did).

It was also removed (via rebase --interactive) from the `permission-api-performance` branch.